ui/src/dashboards/components/variablesControlBar/VariablesControlBar.scss

```diff
@@ -40,3 +39,7 @@ $variables-control-bar--gutter: $ix-marg-a;
 .variables-control-bar > .techno-spinner {
   margin-left: 10px;
 }
+
+.variables-control-bar.presentation-mode {
+  padding-top: 8px !important;
```

Comment:
Usually I try to avoid using `!important` as little as possible. I feel like there's a way to achieve the same look without having to use it